src/globals.ts

```diff
@@ -0,0 +1,41 @@
+declare global {
```

Comment:
Extract the globals to a distinct file for unit-testing and ensuring we've got all them in sync everywhere.